static/js/tippyjs.js

```diff
@@ -179,4 +179,14 @@ export function initialize() {
             instance.destroy();
         },
     });
+
+    delegate("body", {
+        target: ".in_message_time",
```

Comment:
I think this can be `time` (for the HTML element) rather than the class, but we should limit this delegate to `.rendered_markdown` if we're doing that.

Reply:
Yes, that's much better. changed.